packages/core/src/graphql/middleware.ts

```diff
@@ -57,7 +57,7 @@ export const graphql = (
 
       return {
         getDataLoader,
-        getStatus: () => globalThis.PONDER_DATABASE.getStatus(),
+        getCheckpoints: () => globalThis.PONDER_DATABASE.getCheckpoints(),
```

Comment:
I wonder if it would ever make sense to do this using the available drizzle `db` object here rather than the global.